### ISSUE TITLE
Include a notification function that integrates with Slack

### ISSUE THREAD

**Issue by jenniferp96** [rstudio/pointblank#75]
Thanks for making this package! It's very helpful and I got a few validation processes running smoothly in production (no issues with that at all).

I like the email notifier that's included and something along the same lines is a Slack notifier, which would be a great addition! Would that be a feature you're willing to add in?

**Comment by rich-iannone** [comment]
I can do it! Thanks for the feature request!

**Comment by bronsonpetersa** [comment]
Thank you for making this feature a possibility. I would use this a lot.

**Comment by jenniferp96** [comment]
Thanks! Looking forward to this!

**Comment by kylelundstedt** [comment]
Hi @rich-iannone,

Just an FYI - it looks like [the `slackr` package](https://mrkaye97.github.io/slackr/) is shaping up quite nicely.  Perhaps it will be helpful in your work on this issue?

Thanks again for `pointblank`!

**Comment by rich-iannone** [comment]
@kylelundstedt absolutely, and I was waiting on this issue to see how far `slackr` would be progressing in its rewrites. I think a new version hit CRAN recently so I’ll test it out after the next release of `pointblank`. If it works well, I’ll create an adapter function that uses `slacker` for notifications.
